modules/concept-docs/pages/analytics-for-sdk-users.adoc

```diff
@@ -1,5 +1,5 @@
 = Analytics
-:description: Parallel data management for complex queries over many records, using a familiar N1QL-like syntax.
+:description: Parallel data management for complex queries over many records, using a syntax similar to {sqlpp} (formerly N1QL).
```

Comment:
I think this needs a reword, now that they are both SQL++

@simon-dew What do you think?

Maybe we can say "using a familiar {sqlpp} syntax"
It's not, in any case, a good place for the "formerly N1QL" snippet.

Reply:
I agree in this case -- `using a familiar {sqlpp} syntax`